docs/tutorials/tutorial-installing-flox-from-its-repository-on-debian-and-red-hat.mdx

```diff
@@ -0,0 +1,187 @@
+This document describes a practical method for installing Flox from its repositories. Depending on your requirements, this approach may make it easier to create and manage scripts to install Flox on local systems or build it into containers. In addition, systems configured using this method will receive Flox updates as they are released.
+
+When you [download and install a Flox `.deb` or `.rpm` package](https://flox.dev/docs/install-flox/), the installer performs these steps for you. But you can easily script the steps below to automate this process.
+
+These instructions cover the process of installing from Flox’s repository on modern Debian- and Red Hat-based Linux distributions, such as Ubuntu and Linux Mint (Debian), or Fedora, RHEL, and Rocky Linux (Red Hat). They also include an alternate method for installing Flox on Linux systems, such as older versions of CentOS, where `yum` is the default package manager.
+
+
+## 1. Installation overview
+
+Adding Flox's repository to a Debian- or Red Hat-based Linux system’s sources involves the following steps:
+
+-  Importing Flox’s GPG keys to verify authenticity;
+-  Configuring the system’s package manager to recognize Flox’s repository; and
+-  Installing Flox.
+
+
+### Debian-based distros
+
+To perform these steps on a Debian-based distribution using `apt`, do the following.
+
+First, download and import Flox’s GPG keyring.
+
+```
+wget -qO - https://downloads.flox.dev/by-env/stable/deb/flox-archive-keyring.gpg | sudo tee /etc/apt/trusted.gpg.d/flox.gpg >/dev/null
+```
+
+Next, add the Flox repository to ‘/etc/apt/sources.list.d/flox.list’.
+
+```
+sudo add-apt-repository -S "deb https://downloads.flox.dev/by-env/stable/deb/ stable/"
+```
+
+Finally, update[1] and install Flox.
+
+```
+sudo apt update && sudo apt install flox -y
+```
+
+
+### Red Hat-based distros
+
+To perform these steps on modern Red Hat-based distributions that use `dnf`, do the following:
+
+First, download Flox’s GPG keyring.
+
+```
+sudo rpm --import https://downloads.flox.dev/by-env/stable/rpm/flox-archive-keyring.asc
+```
+
+Next, add Flox’s repository to ‘/etc/yum.repos.d/flox.repo’.
+
+```
+sudo dnf config-manager --add-repo https://downloads.flox.dev/by-env/stable/rpm
+```
+
+Finally, install Flox.
+
+``` 
+sudo dnf install flox
+```
+
+
+## 2. Installing from Flox's repository  using “yum”
```

Comment:
I don't know how much we need to cover yum. RHEL8 came out in 2019 and had `dnf`. RHEL7 is on "pay us stupid amounts of money to keep supporting it for 3 more years" zone now.  And all Fedora systems are `dnf` now.